src/RazorSdk/EmbeddedContent/EmbeddedAttribute.cs

```diff
@@ -0,0 +1,8 @@
+// <auto-generated/>
+namespace Microsoft.CodeAnalysis
+{
+    [global::System.AttributeUsage(global::System.AttributeTargets.All, AllowMultiple = true, Inherited = false)]
+    internal sealed class EmbeddedAttribute : global::System.Attribute
```

Comment:
This should be partial, I think?

The AttributeUsage above is probably also unnecessary. I would simply follow what roslyn does here:

https://github.com/dotnet/roslyn/blob/93efd6df1781488d8781f2f9c89d184dcd54ef0b/src/Compilers/CSharp/Portable/SourceGeneration/CSharpGeneratorDriver.cs#L79-L86